hedera-node/src/test/java/com/hedera/services/txns/crypto/CryptoDeleteTransitionLogicTest.java

```diff
@@ -287,9 +302,14 @@ private void givenDeleteTxnMissingTransfer() {
 
 	private TransactionID ourTxnId() {
 		return TransactionID.newBuilder()
-				.setAccountID(payer)
+				.setAccountID(beneficiaryId)
 				.setTransactionValidStart(
 						Timestamp.newBuilder().setSeconds(Instant.now().getEpochSecond()))
 				.build();
 	}
+
+	private void assertFailsWith(Runnable something, ResponseCodeEnum status) {
+		var ex = assertThrows(InvalidTransactionException.class, something::run);
+		assertEquals(status, ex.getResponseCode());
+	}
```

Comment:
You can safely delete this method, as It's now present in `TxnUtils` (or something similar)